tests/ttnn/python_api_testing/non_working_unit_tests/wormhole/test_selu.py

```diff
@@ -0,0 +1,88 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+from loguru import logger
+import random
+import pytest
+import torch
+import ttnn
+import traceback
+
+from tests.ttnn.utils_for_testing import assert_with_pcc
+from tests.ttnn.python_api_testing.sweep_tests import ttnn_ops
+
+
+def run_eltwise_exp_tests(
```

Comment:
Please add the GH Issue number this tests is exposing as a failure in a comment.